moveit_configs_utils/moveit_configs_utils/launches.py

```diff
@@ -219,8 +219,8 @@ def generate_move_group_launch(moveit_config):
     }
 
     move_group_params = [
-        move_group_configuration,
         moveit_config.to_dict(),
+        move_group_configuration,
```

Comment:
Why is the reordering necessary? Is it because we have the parameter in both variables and the second one will be used?

Reply:
I believe so.

Reply:
Should we remove https://github.com/ros-planning/moveit2/blob/7427a8ead23955f69bf9e530bbd06e93f8dcfae0/moveit_configs_utils/moveit_configs_utils/launches.py#L205 then?

Reply:
No, I want that parameter to be set to true.